[2025-08-10T19:48:53.840+0000] {processor.py:186} INFO - Started process (PID=59) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:48:53.841+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:48:53.844+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:48:53.844+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:48:55.228+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:48:55.225+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:48:55.229+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:48:55.247+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 1.417 seconds
[2025-08-10T19:49:26.045+0000] {processor.py:186} INFO - Started process (PID=72) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:49:26.047+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:49:26.049+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:49:26.049+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:49:26.692+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:49:26.690+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:49:26.692+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:49:26.709+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.670 seconds
[2025-08-10T19:49:57.253+0000] {processor.py:186} INFO - Started process (PID=85) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:49:57.254+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:49:57.257+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:49:57.256+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:49:57.914+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:49:57.913+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:49:57.915+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:49:57.932+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.685 seconds
[2025-08-10T19:50:28.450+0000] {processor.py:186} INFO - Started process (PID=99) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:50:28.452+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:50:28.455+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:50:28.454+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:50:29.118+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:50:29.116+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:50:29.119+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:50:29.139+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.695 seconds
[2025-08-10T19:50:59.659+0000] {processor.py:186} INFO - Started process (PID=112) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:50:59.660+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:50:59.662+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:50:59.662+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:51:00.303+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:51:00.302+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:51:00.304+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:51:00.320+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.668 seconds
[2025-08-10T19:51:30.471+0000] {processor.py:186} INFO - Started process (PID=125) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:51:30.473+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:51:30.475+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:51:30.475+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:51:31.109+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:51:31.108+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:51:31.110+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:51:31.126+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.662 seconds
[2025-08-10T19:52:01.890+0000] {processor.py:186} INFO - Started process (PID=138) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:52:01.891+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:52:01.894+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:52:01.893+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:52:02.371+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:52:02.370+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:52:02.372+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:52:02.394+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.512 seconds
[2025-08-10T19:52:33.139+0000] {processor.py:186} INFO - Started process (PID=150) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:52:33.157+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:52:33.160+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:52:33.160+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:52:33.628+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:52:33.626+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:52:33.628+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:52:33.647+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.515 seconds
[2025-08-10T19:53:04.321+0000] {processor.py:186} INFO - Started process (PID=163) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:53:04.322+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:53:04.326+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:53:04.325+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:53:04.800+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:53:04.799+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:53:04.801+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:53:04.821+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.507 seconds
[2025-08-10T19:53:35.552+0000] {processor.py:186} INFO - Started process (PID=176) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:53:35.554+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:53:35.557+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:53:35.557+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:53:36.040+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:53:36.039+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:53:36.041+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:53:36.062+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.517 seconds
[2025-08-10T19:54:06.746+0000] {processor.py:186} INFO - Started process (PID=189) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:54:06.753+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:54:06.756+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:54:06.756+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:54:07.234+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:54:07.233+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:54:07.235+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:54:07.259+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.520 seconds
[2025-08-10T19:54:37.995+0000] {processor.py:186} INFO - Started process (PID=202) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:54:37.996+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:54:37.999+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:54:37.999+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:54:38.477+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:54:38.476+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:54:38.478+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:54:38.500+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.513 seconds
[2025-08-10T19:55:09.180+0000] {processor.py:186} INFO - Started process (PID=215) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:55:09.182+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:55:09.185+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:55:09.184+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:55:09.662+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:55:09.660+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:55:09.662+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:55:09.683+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.510 seconds
[2025-08-10T19:55:40.389+0000] {processor.py:186} INFO - Started process (PID=228) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:55:40.391+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:55:40.394+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:55:40.393+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:55:40.871+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:55:40.869+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:55:40.871+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:55:40.891+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.509 seconds
[2025-08-10T19:56:11.641+0000] {processor.py:186} INFO - Started process (PID=241) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:56:11.642+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:56:11.645+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:56:11.645+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:56:12.127+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:56:12.125+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:56:12.127+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:56:12.147+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.514 seconds
[2025-08-10T19:56:42.831+0000] {processor.py:186} INFO - Started process (PID=254) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:56:42.833+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:56:42.836+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:56:42.836+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:56:43.312+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:56:43.311+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:56:43.313+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:56:43.336+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.513 seconds
[2025-08-10T19:57:14.090+0000] {processor.py:186} INFO - Started process (PID=267) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:57:14.091+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:57:14.095+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:57:14.094+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:57:14.584+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:57:14.582+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:57:14.585+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:57:14.606+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.523 seconds
[2025-08-10T19:57:45.282+0000] {processor.py:186} INFO - Started process (PID=280) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:57:45.284+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:57:45.287+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:57:45.287+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:57:45.764+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:57:45.763+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:57:45.765+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:57:45.787+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.512 seconds
[2025-08-10T19:58:16.477+0000] {processor.py:186} INFO - Started process (PID=293) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:58:16.478+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:58:16.481+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:58:16.481+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:58:16.953+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:58:16.951+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:58:16.953+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:58:16.972+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.502 seconds
[2025-08-10T19:58:47.672+0000] {processor.py:186} INFO - Started process (PID=306) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:58:47.675+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:58:47.678+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:58:47.678+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:58:48.151+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:58:48.150+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:58:48.152+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:58:48.171+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.506 seconds
[2025-08-10T19:59:18.917+0000] {processor.py:186} INFO - Started process (PID=319) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:59:18.918+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:59:18.921+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:59:18.920+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:59:19.395+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:59:19.393+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:59:19.395+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:59:19.416+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.506 seconds
[2025-08-10T19:59:50.095+0000] {processor.py:186} INFO - Started process (PID=332) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:59:50.097+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T19:59:50.100+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:59:50.099+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:59:50.568+0000] {logging_mixin.py:190} INFO - [2025-08-10T19:59:50.566+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T19:59:50.568+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T19:59:50.588+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.500 seconds
[2025-08-10T20:00:21.340+0000] {processor.py:186} INFO - Started process (PID=344) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:00:21.342+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:00:21.345+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:00:21.345+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:00:21.811+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:00:21.810+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:00:21.812+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:00:21.831+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.497 seconds
[2025-08-10T20:00:52.589+0000] {processor.py:186} INFO - Started process (PID=357) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:00:52.591+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:00:52.594+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:00:52.593+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:00:53.066+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:00:53.064+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:00:53.066+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:00:53.086+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.504 seconds
[2025-08-10T20:01:23.773+0000] {processor.py:186} INFO - Started process (PID=370) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:01:23.774+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:01:23.777+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:01:23.777+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:01:24.252+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:01:24.251+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:01:24.253+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:01:24.273+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.507 seconds
[2025-08-10T20:01:55.030+0000] {processor.py:186} INFO - Started process (PID=389) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:01:55.032+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:01:55.035+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:01:55.035+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:01:55.507+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:01:55.505+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:01:55.507+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:01:55.526+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.503 seconds
[2025-08-10T20:02:26.224+0000] {processor.py:186} INFO - Started process (PID=402) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:02:26.225+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:02:26.228+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:02:26.228+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:02:26.700+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:02:26.698+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:02:26.700+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:02:26.724+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.507 seconds
[2025-08-10T20:02:57.411+0000] {processor.py:186} INFO - Started process (PID=415) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:02:57.413+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:02:57.416+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:02:57.416+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:02:57.887+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:02:57.886+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:02:57.888+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:02:57.909+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.505 seconds
[2025-08-10T20:03:28.595+0000] {processor.py:186} INFO - Started process (PID=428) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:03:28.596+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:03:28.599+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:03:28.599+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:03:29.069+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:03:29.067+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:03:29.069+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:03:29.089+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.501 seconds
[2025-08-10T20:03:59.849+0000] {processor.py:186} INFO - Started process (PID=441) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:03:59.851+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:03:59.854+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:03:59.854+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:04:00.331+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:04:00.330+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:04:00.332+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:04:00.349+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.507 seconds
[2025-08-10T20:04:31.090+0000] {processor.py:186} INFO - Started process (PID=455) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:04:31.090+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:04:31.093+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:04:31.093+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:04:31.560+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:04:31.558+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:04:31.560+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:04:31.578+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.496 seconds
[2025-08-10T20:05:02.336+0000] {processor.py:186} INFO - Started process (PID=468) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:05:02.338+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:05:02.341+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:05:02.341+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:05:02.840+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:05:02.838+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:05:02.840+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:05:02.860+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.534 seconds
[2025-08-10T20:05:33.552+0000] {processor.py:186} INFO - Started process (PID=481) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:05:33.554+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:05:33.557+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:05:33.556+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:05:34.037+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:05:34.036+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:05:34.038+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:05:34.058+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.513 seconds
[2025-08-10T20:06:04.802+0000] {processor.py:186} INFO - Started process (PID=494) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:06:04.805+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:06:04.808+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:06:04.807+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:06:05.287+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:06:05.285+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:06:05.288+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:06:05.311+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-10T20:06:35.982+0000] {processor.py:186} INFO - Started process (PID=507) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:06:35.984+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:06:35.987+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:06:35.986+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:06:36.486+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:06:36.484+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:06:36.486+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:06:36.507+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.533 seconds
[2025-08-10T20:07:07.199+0000] {processor.py:186} INFO - Started process (PID=520) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:07:07.200+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:07:07.203+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:07:07.203+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:07:07.682+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:07:07.680+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:07:07.682+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:07:07.703+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.511 seconds
[2025-08-10T20:07:38.458+0000] {processor.py:186} INFO - Started process (PID=533) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:07:38.460+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:07:38.463+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:07:38.463+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:07:38.930+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:07:38.929+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:07:38.931+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:07:38.951+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.500 seconds
[2025-08-10T20:08:09.723+0000] {processor.py:186} INFO - Started process (PID=546) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:08:09.724+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:08:09.727+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:08:09.727+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:08:10.199+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:08:10.197+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:08:10.199+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:08:10.218+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.502 seconds
[2025-08-10T20:08:40.959+0000] {processor.py:186} INFO - Started process (PID=559) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:08:40.961+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:08:40.964+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:08:40.964+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:08:41.440+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:08:41.439+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:08:41.441+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:08:41.460+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.508 seconds
[2025-08-10T20:09:12.195+0000] {processor.py:186} INFO - Started process (PID=572) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:09:12.197+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:09:12.199+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:09:12.199+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:09:12.670+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:09:12.668+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:09:12.670+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:09:12.689+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.501 seconds
[2025-08-10T20:09:43.420+0000] {processor.py:186} INFO - Started process (PID=585) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:09:43.422+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:09:43.425+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:09:43.424+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:09:43.906+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:09:43.904+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:09:43.907+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:09:43.926+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.514 seconds
[2025-08-10T20:10:14.611+0000] {processor.py:186} INFO - Started process (PID=598) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:10:14.612+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:10:14.616+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:10:14.615+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:10:15.093+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:10:15.092+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:10:15.094+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:10:15.114+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.510 seconds
[2025-08-10T20:10:45.860+0000] {processor.py:186} INFO - Started process (PID=611) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:10:45.862+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:10:45.865+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:10:45.864+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:10:46.340+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:10:46.338+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:10:46.340+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:10:46.360+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.507 seconds
[2025-08-10T20:11:17.046+0000] {processor.py:186} INFO - Started process (PID=624) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:11:17.048+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:11:17.051+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:11:17.050+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:11:17.523+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:11:17.522+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:11:17.524+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:11:17.544+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.506 seconds
[2025-08-10T20:11:48.249+0000] {processor.py:186} INFO - Started process (PID=637) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:11:48.251+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:11:48.254+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:11:48.253+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:11:48.744+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:11:48.742+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:11:48.744+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:11:48.761+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.521 seconds
[2025-08-10T20:12:19.451+0000] {processor.py:186} INFO - Started process (PID=650) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:12:19.452+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:12:19.455+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:12:19.454+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:12:19.924+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:12:19.922+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:12:19.924+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:12:19.943+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.500 seconds
[2025-08-10T20:12:50.681+0000] {processor.py:186} INFO - Started process (PID=663) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:12:50.682+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:12:50.685+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:12:50.685+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:12:51.161+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:12:51.160+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:12:51.162+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:12:51.181+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.508 seconds
[2025-08-10T20:13:21.863+0000] {processor.py:186} INFO - Started process (PID=676) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:13:21.864+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:13:21.867+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:13:21.866+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:13:22.342+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:13:22.341+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:13:22.343+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:13:22.363+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.508 seconds
[2025-08-10T20:13:53.138+0000] {processor.py:186} INFO - Started process (PID=689) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:13:53.139+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:13:53.143+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:13:53.142+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:13:53.628+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:13:53.627+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:13:53.629+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:13:53.650+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.520 seconds
[2025-08-10T20:14:24.351+0000] {processor.py:186} INFO - Started process (PID=702) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:14:24.352+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:14:24.356+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:14:24.355+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:14:24.846+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:14:24.844+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:14:24.846+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:14:24.866+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.522 seconds
[2025-08-10T20:14:55.548+0000] {processor.py:186} INFO - Started process (PID=715) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:14:55.550+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:14:55.553+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:14:55.552+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:14:56.032+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:14:56.031+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:14:56.033+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:14:56.056+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-10T20:15:26.763+0000] {processor.py:186} INFO - Started process (PID=734) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:15:26.764+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:15:26.767+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:15:26.766+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:15:27.249+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:15:27.247+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:15:27.249+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:15:27.274+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.519 seconds
[2025-08-10T20:15:57.964+0000] {processor.py:186} INFO - Started process (PID=747) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:15:57.966+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:15:57.969+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:15:57.968+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:15:58.443+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:15:58.442+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:15:58.444+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:15:58.465+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.508 seconds
[2025-08-10T20:16:29.179+0000] {processor.py:186} INFO - Started process (PID=760) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:16:29.183+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:16:29.186+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:16:29.186+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:16:29.666+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:16:29.665+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:16:29.667+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:16:29.687+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.515 seconds
[2025-08-10T20:17:00.376+0000] {processor.py:186} INFO - Started process (PID=773) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:17:00.378+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:17:00.381+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:17:00.380+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:17:00.865+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:17:00.863+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:17:00.865+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:17:00.885+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.517 seconds
[2025-08-10T20:17:31.652+0000] {processor.py:186} INFO - Started process (PID=786) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:17:31.654+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:17:31.657+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:17:31.657+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:17:32.131+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:17:32.129+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:17:32.131+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:17:32.151+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.506 seconds
[2025-08-10T20:18:02.921+0000] {processor.py:186} INFO - Started process (PID=799) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:18:02.922+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:18:02.925+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:18:02.924+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:18:03.412+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:18:03.411+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:18:03.413+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:18:03.433+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.520 seconds
[2025-08-10T20:18:34.117+0000] {processor.py:186} INFO - Started process (PID=812) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:18:34.118+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:18:34.122+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:18:34.121+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:18:34.603+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:18:34.602+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:18:34.604+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:18:34.624+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.515 seconds
[2025-08-10T20:19:05.432+0000] {processor.py:186} INFO - Started process (PID=825) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:19:05.433+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:19:05.436+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:19:05.436+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:19:05.906+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:19:05.905+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:19:05.907+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:19:05.926+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.502 seconds
[2025-08-10T20:19:36.623+0000] {processor.py:186} INFO - Started process (PID=838) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:19:36.670+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:19:36.673+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:19:36.673+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:19:37.154+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:19:37.152+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:19:37.154+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:19:37.172+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.556 seconds
[2025-08-10T20:20:07.870+0000] {processor.py:186} INFO - Started process (PID=852) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:20:07.871+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:20:07.875+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:20:07.875+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:20:08.345+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:20:08.343+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:20:08.346+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:20:08.363+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.500 seconds
[2025-08-10T20:20:39.116+0000] {processor.py:186} INFO - Started process (PID=865) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:20:39.117+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:20:39.120+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:20:39.120+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:20:39.710+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:20:39.707+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:20:39.710+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:20:39.734+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.625 seconds
[2025-08-10T20:21:10.572+0000] {processor.py:186} INFO - Started process (PID=878) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:21:10.573+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:21:10.577+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:21:10.576+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:21:11.061+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:21:11.060+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:21:11.062+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:21:11.081+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-10T20:21:41.805+0000] {processor.py:186} INFO - Started process (PID=891) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:21:41.806+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:21:41.809+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:21:41.809+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:21:42.281+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:21:42.279+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:21:42.281+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:21:42.300+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.503 seconds
[2025-08-10T20:22:13.040+0000] {processor.py:186} INFO - Started process (PID=904) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:22:13.042+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:22:13.045+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:22:13.045+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:22:13.517+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:22:13.515+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:22:13.517+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:22:13.536+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.503 seconds
[2025-08-10T20:22:44.225+0000] {processor.py:186} INFO - Started process (PID=917) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:22:44.226+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:22:44.229+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:22:44.228+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:22:44.705+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:22:44.704+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:22:44.706+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:22:44.727+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.510 seconds
[2025-08-10T20:23:15.472+0000] {processor.py:186} INFO - Started process (PID=930) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:23:15.474+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:23:15.477+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:23:15.476+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:23:15.948+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:23:15.947+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:23:15.949+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:23:15.973+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.509 seconds
[2025-08-10T20:23:46.690+0000] {processor.py:186} INFO - Started process (PID=943) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:23:46.691+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:23:46.694+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:23:46.693+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:23:47.167+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:23:47.165+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:23:47.168+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:23:47.189+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.506 seconds
[2025-08-10T20:24:17.914+0000] {processor.py:186} INFO - Started process (PID=956) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:24:17.916+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:24:17.919+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:24:17.919+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:24:18.393+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:24:18.392+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:24:18.394+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:24:18.416+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.509 seconds
[2025-08-10T20:24:49.094+0000] {processor.py:186} INFO - Started process (PID=969) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:24:49.095+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:24:49.098+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:24:49.098+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:24:49.573+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:24:49.572+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:24:49.574+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:24:49.596+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.509 seconds
[2025-08-10T20:25:20.360+0000] {processor.py:186} INFO - Started process (PID=982) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:25:20.362+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:25:20.365+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:25:20.364+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:25:20.839+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:25:20.837+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:25:20.840+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:25:20.860+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.507 seconds
[2025-08-10T20:25:51.535+0000] {processor.py:186} INFO - Started process (PID=995) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:25:51.536+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:25:51.539+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:25:51.539+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:25:52.019+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:25:52.018+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:25:52.020+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:25:52.044+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.517 seconds
[2025-08-10T20:26:22.745+0000] {processor.py:186} INFO - Started process (PID=1008) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:26:22.748+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:26:22.751+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:26:22.751+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:26:23.237+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:26:23.235+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:26:23.237+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:26:23.257+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.520 seconds
[2025-08-10T20:26:53.958+0000] {processor.py:186} INFO - Started process (PID=1021) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:26:53.959+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:26:53.962+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:26:53.962+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:26:54.440+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:26:54.439+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:26:54.441+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:26:54.461+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.511 seconds
[2025-08-10T20:27:25.246+0000] {processor.py:186} INFO - Started process (PID=1034) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:27:25.247+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:27:25.251+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:27:25.250+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:27:25.734+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:27:25.732+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:27:25.734+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:27:25.754+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-10T20:27:56.450+0000] {processor.py:186} INFO - Started process (PID=1047) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:27:56.451+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:27:56.454+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:27:56.453+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:27:56.929+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:27:56.927+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:27:56.929+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:27:56.950+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.507 seconds
[2025-08-10T20:28:27.654+0000] {processor.py:186} INFO - Started process (PID=1065) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:28:27.656+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:28:27.660+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:28:27.659+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:28:28.157+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:28:28.156+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:28:28.158+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:28:28.179+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.533 seconds
[2025-08-10T20:28:58.861+0000] {processor.py:186} INFO - Started process (PID=1078) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:28:58.862+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:28:58.865+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:28:58.864+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:28:59.339+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:28:59.338+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:28:59.340+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:28:59.358+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.505 seconds
[2025-08-10T20:29:30.046+0000] {processor.py:186} INFO - Started process (PID=1091) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:29:30.047+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:29:30.051+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:29:30.050+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:29:30.530+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:29:30.529+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:29:30.531+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:29:30.550+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.512 seconds
[2025-08-10T20:30:01.242+0000] {processor.py:186} INFO - Started process (PID=1104) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:30:01.243+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:30:01.246+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:30:01.246+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:30:01.729+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:30:01.728+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:30:01.730+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:30:01.751+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-10T20:30:32.441+0000] {processor.py:186} INFO - Started process (PID=1117) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:30:32.442+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:30:32.445+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:30:32.445+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:30:32.915+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:30:32.914+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:30:32.916+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:30:32.936+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.502 seconds
[2025-08-10T20:31:03.635+0000] {processor.py:186} INFO - Started process (PID=1130) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:31:03.636+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:31:03.639+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:31:03.639+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:31:04.119+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:31:04.118+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:31:04.120+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:31:04.141+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.513 seconds
[2025-08-10T20:31:34.831+0000] {processor.py:186} INFO - Started process (PID=1143) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:31:34.833+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:31:34.836+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:31:34.835+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:31:35.323+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:31:35.322+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:31:35.324+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:31:35.344+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.521 seconds
[2025-08-10T20:32:06.032+0000] {processor.py:186} INFO - Started process (PID=1156) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:32:06.033+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:32:06.036+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:32:06.036+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:32:06.513+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:32:06.511+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:32:06.513+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:32:06.532+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.507 seconds
[2025-08-10T20:32:37.225+0000] {processor.py:186} INFO - Started process (PID=1169) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:32:37.226+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:32:37.229+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:32:37.229+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:32:37.701+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:32:37.700+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:32:37.702+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:32:37.724+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.506 seconds
[2025-08-10T20:33:08.415+0000] {processor.py:186} INFO - Started process (PID=1182) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:33:08.417+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:33:08.420+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:33:08.420+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:33:08.902+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:33:08.901+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:33:08.903+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:33:08.923+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-10T20:33:39.690+0000] {processor.py:186} INFO - Started process (PID=1195) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:33:39.691+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:33:39.694+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:33:39.693+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:33:40.170+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:33:40.168+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:33:40.170+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:33:40.195+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.513 seconds
[2025-08-10T20:34:10.886+0000] {processor.py:186} INFO - Started process (PID=1208) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:34:10.888+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:34:10.891+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:34:10.890+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:34:11.362+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:34:11.360+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:34:11.362+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:34:11.382+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.503 seconds
[2025-08-10T20:34:42.067+0000] {processor.py:186} INFO - Started process (PID=1221) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:34:42.068+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:34:42.071+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:34:42.071+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:34:42.540+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:34:42.539+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:34:42.541+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:34:42.564+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.505 seconds
[2025-08-10T20:35:13.259+0000] {processor.py:186} INFO - Started process (PID=1234) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:35:13.261+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:35:13.264+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:35:13.263+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:35:13.736+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:35:13.734+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:35:13.736+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:35:13.756+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.504 seconds
[2025-08-10T20:35:44.504+0000] {processor.py:186} INFO - Started process (PID=1248) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:35:44.505+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:35:44.507+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:35:44.507+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:35:44.978+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:35:44.977+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:35:44.979+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:35:44.998+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.501 seconds
[2025-08-10T20:36:15.756+0000] {processor.py:186} INFO - Started process (PID=1261) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:36:15.758+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:36:15.761+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:36:15.761+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:36:16.235+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:36:16.234+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:36:16.236+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:36:16.255+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.506 seconds
[2025-08-10T20:37:58.288+0000] {processor.py:186} INFO - Started process (PID=61) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:37:58.291+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:37:58.296+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:37:58.295+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:37:59.537+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:37:59.535+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:37:59.537+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:37:59.554+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 1.278 seconds
[2025-08-10T20:38:30.527+0000] {processor.py:186} INFO - Started process (PID=74) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:38:30.529+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:38:30.531+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:38:30.531+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:38:31.218+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:38:31.217+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:38:31.219+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:38:31.236+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.715 seconds
[2025-08-10T20:39:01.699+0000] {processor.py:186} INFO - Started process (PID=93) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:39:01.700+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:39:01.703+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:39:01.702+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:39:02.367+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:39:02.366+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:39:02.368+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:39:02.383+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.691 seconds
[2025-08-10T20:39:32.891+0000] {processor.py:186} INFO - Started process (PID=106) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:39:32.893+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:39:32.896+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:39:32.895+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:39:33.539+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:39:33.538+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:39:33.540+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:39:33.557+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.672 seconds
[2025-08-10T20:40:04.471+0000] {processor.py:186} INFO - Started process (PID=119) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:40:04.471+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:40:04.474+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:40:04.474+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:40:05.126+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:40:05.124+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:40:05.126+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:40:05.142+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.678 seconds
[2025-08-10T20:40:35.673+0000] {processor.py:186} INFO - Started process (PID=132) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:40:35.676+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:40:35.856+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:40:35.679+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:40:36.317+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:40:36.316+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:40:36.317+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:40:36.334+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.667 seconds
[2025-08-10T20:41:07.124+0000] {processor.py:186} INFO - Started process (PID=145) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:41:07.125+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:41:07.127+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:41:07.127+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:41:07.616+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:41:07.615+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:41:07.617+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:41:07.639+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.524 seconds
[2025-08-10T20:41:38.338+0000] {processor.py:186} INFO - Started process (PID=158) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:41:38.341+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:41:38.343+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:41:38.343+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:41:38.819+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:41:38.818+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:41:38.820+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:41:38.839+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.509 seconds
[2025-08-10T20:42:09.617+0000] {processor.py:186} INFO - Started process (PID=171) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:42:09.619+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:42:09.622+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:42:09.622+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:42:10.102+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:42:10.100+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:42:10.102+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:42:10.125+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-10T20:42:40.877+0000] {processor.py:186} INFO - Started process (PID=184) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:42:40.879+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:42:40.882+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:42:40.881+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:42:41.374+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:42:41.372+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:42:41.374+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:42:41.395+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.526 seconds
[2025-08-10T20:43:12.185+0000] {processor.py:186} INFO - Started process (PID=197) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:43:12.186+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:43:12.190+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:43:12.189+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:43:12.685+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:43:12.684+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:43:12.686+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:43:12.705+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.527 seconds
[2025-08-10T20:43:43.396+0000] {processor.py:186} INFO - Started process (PID=210) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:43:43.398+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:43:43.401+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:43:43.400+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:43:43.908+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:43:43.906+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:43:43.908+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:43:43.928+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.539 seconds
[2025-08-10T20:44:14.607+0000] {processor.py:186} INFO - Started process (PID=223) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:44:14.608+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:44:14.611+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:44:14.610+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:44:15.103+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:44:15.102+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:44:15.104+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:44:15.123+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.524 seconds
[2025-08-10T20:44:45.818+0000] {processor.py:186} INFO - Started process (PID=236) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:44:45.821+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:44:45.824+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:44:45.824+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:44:46.310+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:44:46.308+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:44:46.310+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:44:46.332+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.521 seconds
[2025-08-10T20:45:17.034+0000] {processor.py:186} INFO - Started process (PID=249) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:45:17.035+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:45:17.038+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:45:17.037+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:45:17.527+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:45:17.525+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:45:17.527+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:45:17.548+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.522 seconds
[2025-08-10T20:45:48.248+0000] {processor.py:186} INFO - Started process (PID=262) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:45:48.249+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:45:48.253+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:45:48.252+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:45:48.730+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:45:48.729+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:45:48.731+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:45:48.750+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.511 seconds
[2025-08-10T20:46:19.457+0000] {processor.py:186} INFO - Started process (PID=275) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:46:19.458+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:46:19.461+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:46:19.461+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:46:19.967+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:46:19.965+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:46:19.968+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:46:19.992+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.542 seconds
[2025-08-10T20:46:50.669+0000] {processor.py:186} INFO - Started process (PID=288) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:46:50.671+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:46:50.674+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:46:50.674+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:46:51.160+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:46:51.159+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:46:51.161+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:46:51.183+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.522 seconds
[2025-08-10T20:47:21.873+0000] {processor.py:186} INFO - Started process (PID=301) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:47:21.875+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:47:21.878+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:47:21.877+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:47:22.377+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:47:22.376+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:47:22.378+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:47:22.401+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.536 seconds
[2025-08-10T20:47:53.074+0000] {processor.py:186} INFO - Started process (PID=314) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:47:53.076+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:47:53.079+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:47:53.078+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:47:53.567+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:47:53.566+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:47:53.568+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:47:53.593+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.526 seconds
[2025-08-10T20:48:24.297+0000] {processor.py:186} INFO - Started process (PID=327) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:48:24.298+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:48:24.301+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:48:24.301+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:48:24.798+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:48:24.797+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:48:24.799+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:48:24.819+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.530 seconds
[2025-08-10T20:48:55.502+0000] {processor.py:186} INFO - Started process (PID=340) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:48:55.504+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:48:55.507+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:48:55.507+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:48:56.000+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:48:55.999+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:48:56.001+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:48:56.022+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.528 seconds
[2025-08-10T20:49:26.706+0000] {processor.py:186} INFO - Started process (PID=353) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:49:26.707+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:49:26.710+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:49:26.709+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:49:27.197+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:49:27.195+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:49:27.197+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:49:27.218+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.519 seconds
[2025-08-10T20:49:57.910+0000] {processor.py:186} INFO - Started process (PID=366) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:49:57.912+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:49:57.915+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:49:57.914+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:49:58.397+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:49:58.395+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:49:58.397+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:49:58.418+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-10T20:50:29.230+0000] {processor.py:186} INFO - Started process (PID=379) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:50:29.231+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:50:29.234+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:50:29.234+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:50:29.736+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:50:29.734+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:50:29.737+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:50:29.758+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.536 seconds
[2025-08-10T20:51:00.429+0000] {processor.py:186} INFO - Started process (PID=392) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:51:00.483+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:51:00.486+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:51:00.485+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:51:00.984+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:51:00.982+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:51:00.984+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:51:01.004+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.582 seconds
[2025-08-10T20:51:31.628+0000] {processor.py:186} INFO - Started process (PID=405) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:51:31.629+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:51:31.632+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:51:31.631+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:51:32.116+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:51:32.114+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:51:32.117+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:51:32.136+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-10T20:52:02.943+0000] {processor.py:186} INFO - Started process (PID=424) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:52:02.945+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:52:02.948+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:52:02.948+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:52:03.440+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:52:03.439+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:52:03.441+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:52:03.462+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.526 seconds
[2025-08-10T20:52:34.142+0000] {processor.py:186} INFO - Started process (PID=437) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:52:34.145+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:52:34.149+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:52:34.149+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:52:34.641+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:52:34.640+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:52:34.642+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:52:34.663+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.528 seconds
[2025-08-10T20:53:05.402+0000] {processor.py:186} INFO - Started process (PID=451) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:53:05.403+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:53:05.407+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:53:05.406+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:53:05.910+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:53:05.909+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:53:05.911+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:53:05.933+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.538 seconds
[2025-08-10T20:53:36.725+0000] {processor.py:186} INFO - Started process (PID=464) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:53:36.726+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:53:36.729+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:53:36.729+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:53:37.223+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:53:37.222+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:53:37.224+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:53:37.246+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.528 seconds
[2025-08-10T20:54:07.948+0000] {processor.py:186} INFO - Started process (PID=477) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:54:07.950+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:54:07.954+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:54:07.953+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:54:08.486+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:54:08.485+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:54:08.487+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:54:08.513+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.572 seconds
[2025-08-10T20:54:39.327+0000] {processor.py:186} INFO - Started process (PID=490) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:54:39.328+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:54:39.331+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:54:39.331+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:54:39.816+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:54:39.815+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:54:39.817+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:54:39.836+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-10T20:55:10.615+0000] {processor.py:186} INFO - Started process (PID=503) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:55:10.617+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:55:10.621+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:55:10.621+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:55:11.187+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:55:11.185+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:55:11.188+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:55:11.210+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.604 seconds
[2025-08-10T20:55:41.831+0000] {processor.py:186} INFO - Started process (PID=517) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:55:41.832+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:55:41.835+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:55:41.835+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:55:42.319+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:55:42.318+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:55:42.320+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:55:42.343+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.519 seconds
[2025-08-10T20:56:13.048+0000] {processor.py:186} INFO - Started process (PID=530) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:56:13.050+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:56:13.053+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:56:13.053+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:56:13.554+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:56:13.553+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:56:13.555+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:56:13.577+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.538 seconds
[2025-08-10T20:56:44.261+0000] {processor.py:186} INFO - Started process (PID=543) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:56:44.262+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:56:44.265+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:56:44.265+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:56:44.762+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:56:44.761+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:56:44.763+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:56:44.782+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.528 seconds
[2025-08-10T20:57:15.491+0000] {processor.py:186} INFO - Started process (PID=556) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:57:15.493+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:57:15.496+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:57:15.496+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:57:15.990+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:57:15.988+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:57:15.990+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:57:16.009+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.525 seconds
[2025-08-10T20:57:46.769+0000] {processor.py:186} INFO - Started process (PID=569) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:57:46.770+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:57:46.773+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:57:46.773+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:57:47.281+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:57:47.280+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:57:47.282+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:57:47.306+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.545 seconds
[2025-08-10T20:58:18.223+0000] {processor.py:186} INFO - Started process (PID=582) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:58:18.224+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:58:18.228+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:58:18.228+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:58:18.726+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:58:18.725+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:58:18.727+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:58:18.751+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.537 seconds
[2025-08-10T20:58:49.427+0000] {processor.py:186} INFO - Started process (PID=595) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:58:49.428+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:58:49.431+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:58:49.431+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:58:49.938+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:58:49.937+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:58:49.939+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:58:49.959+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.541 seconds
[2025-08-10T20:59:20.723+0000] {processor.py:186} INFO - Started process (PID=608) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:59:20.725+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:59:20.731+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:59:20.731+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:59:21.259+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:59:21.257+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:59:21.260+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:59:21.281+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.565 seconds
[2025-08-10T20:59:52.104+0000] {processor.py:186} INFO - Started process (PID=621) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:59:52.113+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T20:59:52.116+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:59:52.116+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:59:52.615+0000] {logging_mixin.py:190} INFO - [2025-08-10T20:59:52.614+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T20:59:52.616+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T20:59:52.637+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.541 seconds
[2025-08-10T21:00:23.321+0000] {processor.py:186} INFO - Started process (PID=634) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:00:23.322+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T21:00:23.325+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:00:23.324+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:00:23.816+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:00:23.814+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T21:00:23.816+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:00:23.840+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.527 seconds
[2025-08-10T21:00:54.534+0000] {processor.py:186} INFO - Started process (PID=647) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:00:54.541+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T21:00:54.544+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:00:54.544+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:00:55.057+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:00:55.055+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T21:00:55.058+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:00:55.077+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.552 seconds
[2025-08-10T21:01:25.753+0000] {processor.py:186} INFO - Started process (PID=660) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:01:25.758+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T21:01:25.761+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:01:25.760+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:01:26.251+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:01:26.249+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T21:01:26.251+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:01:26.432+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.687 seconds
[2025-08-10T21:01:56.484+0000] {processor.py:186} INFO - Started process (PID=673) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:01:56.535+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T21:01:56.539+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:01:56.538+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:01:57.048+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:01:57.046+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T21:01:57.049+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:01:57.074+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.599 seconds
[2025-08-10T21:02:27.927+0000] {processor.py:186} INFO - Started process (PID=686) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:02:27.936+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T21:02:27.939+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:02:27.939+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:02:28.435+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:02:28.434+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T21:02:28.436+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:02:28.455+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.535 seconds
[2025-08-10T21:02:59.153+0000] {processor.py:186} INFO - Started process (PID=699) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:02:59.154+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T21:02:59.157+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:02:59.157+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:02:59.645+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:02:59.644+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T21:02:59.646+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:02:59.666+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.520 seconds
[2025-08-10T21:03:30.359+0000] {processor.py:186} INFO - Started process (PID=712) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:03:30.365+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T21:03:30.368+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:03:30.368+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:03:30.870+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:03:30.868+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T21:03:30.870+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:03:30.891+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.540 seconds
[2025-08-10T21:04:01.678+0000] {processor.py:186} INFO - Started process (PID=725) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:04:01.679+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-10T21:04:01.683+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:04:01.683+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:04:02.423+0000] {logging_mixin.py:190} INFO - [2025-08-10T21:04:02.421+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-10T21:04:02.424+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-10T21:04:02.448+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.778 seconds
